lib/protocol/hpack/context.rb

```diff
@@ -296,6 +301,11 @@ def add_to_table(command)
 				command.freeze
 				
 				@table.unshift(command)
+				@cursize += entry_size(command)
```

Comment:
Small nit, if we are going to introduce this, can we rename it to `@current_table_size`?